halo2_proofs/README.md

```diff
@@ -15,6 +15,9 @@ minor version bump.
 computation. The `RAYON_NUM_THREADS` environment variable can be used to set the number of
 threads.
 
+You can disable [rayon] by disabling the ["multicore"] feature.
+Warning! Halo2 will lose access to parallelism if you disable the ["multicore"] feature. This will significantly downgrade the performance. 
```

Comment:
```suggestion
Warning! Halo2 will lose access to parallelism if you disable the ["multicore"] feature.
This will significantly degrade the performance. 
```